pytube/extract.py

```diff
@@ -396,3 +398,48 @@ def apply_descrambler(stream_data: Dict, key: str) -> None:
         ]
 
     logger.debug("applying descrambler")
+
+
+def initial_data(watch_html: str) -> str:
+    """
+    Extracts the initial_data json from the watch_html page.
```

Comment:
Extracts the initial_data json from the watch_html page. ->     Extracts a json from youtube watch_html page called "initial_data" that contains metadata about the video used by other functions such as copyright information.